src/Parsers/ParserCreateQuery.cpp

```diff
@@ -818,6 +878,7 @@ bool ParserCreateLiveViewQuery::parseImpl(Pos & pos, ASTPtr & node, Expected & e
     ParserStorage storage_inner{ParserStorage::TABLE_ENGINE};
     ParserTablePropertiesDeclarationList table_properties_p;
     ParserSelectWithUnionQuery select_p;
+    ParserSQLSecurity sql_security_p;
```

Comment:
Should we support this functionality for LiveView?

Reply:
Why not? It works almost like a normal view, so it's easy to add.

Reply:
ok